### DISCUSSION TITLE
Is Metric Expiry stable?

### DISCUSSION THREAD

**Discussion by m-wynn** [istio/istio#49383] [Q&A]
I see that Metric Expiry (implemented: https://github.com/istio/istio/pull/44605 released in 1.18: https://github.com/istio/istio/pull/45037) solves a lot of memory issues in both Istio and Prometheus/VictoriaMetrics for long-running applications with a lot of short-lived metrics.

I'm wondering how people's experiences with setting those environment variables are, if it's close to being considered stable, and if we'll see it in the telemetry API in an upcoming release.

**Comment by zirain** [comment]
we're glad to see it help someone, maybe it's time to make it stable/proposal an API.
